samples/apps/logging/js/src/logging.js

```diff
@@ -15,6 +15,20 @@ function get_id_from_query(parsedQuery) {
   return ccf.strToBuf(parsedQuery.id);
 }
 
+function get_scope(parsedQuery) {
+  return parsedQuery.scope === undefined ? undefined : parsedQuery.scope;
```

Comment:
I'm super confused by this, how is that not equivalent to parsedQuery.scope?

Reply:
I remember having a reason for it, but I agree, it should be totally unnecessary. I think I was just confused about undefined/null/`===`.